ctk/common/src/test/kotlin/org/codice/compilance/verification/profile/SingleSignOnProfileVerifierSpec.kt

```diff
@@ -168,3 +278,5 @@ class SingleSignOnProfileVerifierSpec : StringSpec() {
             """.trimMargin()
     }
 }
+
+class CoreVerifierTest(samlNode: NodeDecorator) : CoreVerifier(samlNode)
```

Comment:
❓ Why is this class being extended if no fields/methods are being overridden? Can you just use the base class instead?

Reply:
Because `CoreVerifier` is abstract so it can't been directly instantiated.